data/hoverboard.config.json

```diff
@@ -55,5 +55,33 @@
     "lg": {
       "min": "1024px"
     }
-  }
+  },
+  "socialNetwork": [
+    {
+      "name": "facebook",
+      "url": "https://www.facebook.com/devfest.ukraine/"
+    },
+    {
+      "name": "twitter",
+      "url": "https://twitter.com/devfest_ukraine/"
+    },
+    {
+      "name": "instagram",
+      "url": "https://www.instagram.com/devfestukraine"
+    },
+    {
+      "name": "youtube",
+      "url": "https://www.youtube.com/playlist?list=PLt8lEzcLNl31AouwDkbLbLARlQFwNS2en"
+    }
+  ],
+  "gplusShareLink": "https://plus.google.com/share?url=https://devfest.gdg.org.ua/",
```

Comment:
Why only G+ no one is using it right now :)
Create a generic behavior that includes functions to share with all social networks and is extendable. See https://github.com/gdg-x/hoverboard/blob/master/src/behaviors/share-behavior.html
Also, don't forget about new features that would be nice to include https://developers.google.com/web/updates/2016/10/navigator-share